Volha_Ivanova/Module_1_Java/Classes_and_Objects/src/com/epam/cdp/volha_ivanova/task1/sweets/candies/chocolateCandies/ChocolateCandies.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.cdp.volha_ivanova.task1.sweets.candies.chocolateCandies;
```

Comment:
packages it's good, but each class shouldn't be in separate package. Find something between)
I provided example of my solution. Please review it and I think my comment will be clear.